src/pages/trade/HorizontalPanel.tsx

```diff
@@ -110,7 +110,11 @@ export const HorizontalPanel = ({ isOpen = true, setIsOpen }: ElementProps) => {
                     PositionsTableColumnKey.UnrealizedPnl,
                     PositionsTableColumnKey.RealizedPnl,
                     PositionsTableColumnKey.AverageOpenAndClose,
-                  ]
+                    ...(testFlags.closePositionsFromPositionsTable ||
+                    testFlags.configureSlTpFromPositionsTable
+                      ? [!isAccountViewOnly && PositionsTableColumnKey.Actions]
```

Comment:
combine `testFlags` and `isAccountViewOnly` into a separate `createSelector` in `accountsSelector` so we don't duplicate logic here and in the `<Positions>` page.